iot_verifier/tests/rewarder_operations.rs

```diff
@@ -0,0 +1,35 @@
+mod common;
+use chrono::{Duration as ChronoDuration, Utc};
+use iot_verifier::{reward_share, rewarder};
+use rust_decimal::{prelude::ToPrimitive, Decimal, RoundingStrategy};
+use rust_decimal_macros::dec;
+use sqlx::PgPool;
+
+#[sqlx::test]
+async fn test_operations(_pool: PgPool) -> anyhow::Result<()> {
```

Comment:
this `PgPool` is not used? if this test isn't an impl of a trait that requires it is this argument not needed anymore?

Reply:
Addressed in sep PR via this commit: d1238dc8e5e255b4aa03f3c4a41fc6da62053b04